sdk/inc/azure/iot/az_iot_provisioning_client.h

```diff
@@ -258,34 +270,19 @@ AZ_NODISCARD az_result az_iot_provisioning_client_parse_received_topic_and_paylo
     az_span received_payload,
     az_iot_provisioning_client_register_response* out_response);
 
-/**
- * @brief Azure IoT Provisioning Service operation status.
- *
- */
-typedef enum
-{
-  // Device assignment in progress.
-  AZ_IOT_PROVISIONING_STATUS_UNASSIGNED,
-  AZ_IOT_PROVISIONING_STATUS_ASSIGNING,
-
-  // Device assignment operation complete.
-  AZ_IOT_PROVISIONING_STATUS_ASSIGNED,
-  AZ_IOT_PROVISIONING_STATUS_FAILED,
-  AZ_IOT_PROVISIONING_STATUS_DISABLED,
-} az_iot_provisioning_client_operation_status;
-
 /**
  * @brief Returns the #az_iot_provisioning_client_operation_status of a
  * #az_iot_provisioning_client_register_response object.
  *
- * @param[in] response The #az_iot_provisioning_client_register_response obtained after a successful
- *                     call to az_iot_provisioning_client_parse_received_topic_and_payload().
+ * @param[in] response_operation_status An #az_span of the operation status obtained 
+ *                     after a successful call to 
+ *                     az_iot_provisioning_client_parse_received_topic_and_payload().
  * @param[out] out_operation_status The registration operation status.
  * @return An #az_result value indicating the result of the operation.
  * @retval #AZ_ERROR_UNEXPECTED_CHAR If the string contains an unexpected value.
  */
 AZ_NODISCARD az_result az_iot_provisioning_client_parse_operation_status(

```

Comment:
I don't think this method needs to be public anymore. Make private static?

Reply:
agreed I don't think this is needed anymore